configuration/analyzer_config.json

```diff
@@ -2816,6 +2816,33 @@
       }
     }
   },
+  "Stalkphish": {
+    "type": "observable",
+    "python_module": "stalkphish.Stalkphish",
+    "description": "Check in url or ipv4 if string exists in [Stalkphish.io](https://www.stalkphish.io/) API.",
+    "maximum_tlp": "RED",
```

Comment:
this value is needed only for `connectors`. Can be removed

Reply:
ok.